airflow/www/views.py

```diff
@@ -2924,6 +2925,16 @@ def tree_data(self):
         # avoid spaces to reduce payload size
         return htmlsafe_json_dumps(tree_data, separators=(',', ':'))
 
+    @expose('/robots.txt')
+    @action_logging
+    def robots(self):
+        """
+        Returns a robots.txt file for blocking certain search engine crawlers. This mitigates some
+        of the risk associated with exposing Airflow to the public internet, however it does not
+        address the real security risks associated with such a deployment.
+        """
+        return send_from_directory(current_app.static_folder, 'robots.txt')
```

Comment:
For such a simple file, it’s probably easier to simply embed `robot.txt` in the view function instead of using `send_from_directory` (which introduces filesystem overhead).

Reply:
easy enough to change, just thought it'd be easier to maintain as a physical file. - wouldn't expect much traffic/load on this endpoint anyway

Reply:
Your opinion on the topic is way stronger than mine, so will amend the PR, for someone not familiar with flask - searching the file-system for a file named "robots.txt" ought to be the easiest way to find and alter the content on that file however.

Reply:
I would agree with @thejens on this. The most common place anyone (not only those familiar with Flask) would expect to find this file is in a static assets directory.

Reply:
Agree with @thejens  too. I'd definitely look for robots.txt file

Reply:
Right, changing it back again :) just waiting for the pre-commit-hook to finish